DRAFT_birdIndicators.Rmd

```diff
@@ -0,0 +1,481 @@
+
+# Relative general and working folders for RSWAN
+general_folder_rel <- "data"
+working_folder_rel <- paste0(folder, "/", subFolderName)
+
+# Storage of MSI results
+MSI_results_folder <- "MSI_Results"
+
+
+## Absolute directory/file paths
+
+# NINAs local storage of legacy Trim output files
+legacyFile_folder <- "P:/41201612_naturindeks_2021_2023_database_og_innsynslosning/Hekkefugl_Dataflyt/LegacyFiles_PECBMS_Trim"
+
+# Absolute data and output paths for RSWAN 
+general_folder <- paste0(getwd(), "/", general_folder_rel)
+working_folder <- paste0(getwd(), "/", working_folder_rel)
+
+output_folder <- paste0(working_folder, "/output/") 
+output_folder2 <- paste0(working_folder, "/output") 
+
+
+## Create directories if they are not available
+dir.list <- c(folder, general_folder_rel, working_folder_rel, output_folder2)
+for(i in 1:length(dir.list)){
+  if(!file.exists(dir.list[i])){ 
+  dir.create(dir.list[i])
+  }
+}
+
+```
+
+### Data sets
+The workflow relies on two major sources of data: the up-to-date raw data from "hekkefuglovervåkingen" and the calculated abundance indices from the predecessor monitoring schemes. Additionally, we also a couple of lists holding information on which species (and years) are to be included in which parts of the analytical workflow.  
+
+#### Monitoring data from "Hekkefuglovervåkingen"
+The route-based raw data from "hekkefuglovervåkingen" are compiled after each breeding season and stored in NINA's database. From that database we download the data relevant for the PECBMS Trim analyses and for the specified range of years (here from 2006 up to and including 2022). 
+
+```{r, eval = FALSE}
+minYear <- 2006
+maxYear <- 2022
+
+Trim_data <- downloadData_TRIM(minYear = minYear, maxYear = maxYear,
+                               drop_negativeSpp = TRUE, DriverName = "FreeTDS")
+```
+
+#### PECBMS Trim results from predecessor monitoring programme
+The abundance indices generated by the PECBMS RTRIM workflow using data from the predecessor scheme are stored in NINAs local file repositories. Unlike the Trim analyses for the data from "hekkefuglovervåkingen", which are re-run every year after new data has been added, the old abundance indices remain unchanged as no new data are being added.
+These data are needed for the RSWAN workflow, and they need to be present in the respective "working_folder". So we copy them from their location in NINA's local file storage into the correct folder for analyses. 
+
+```{r, eval = FALSE}
+collectSpeciesFiles_Legacy(origin_folder = legacyFile_folder,
+                           target_folder = paste0(folder, "/", subFolderName))
+``` 
+
+#### Lists of species information
+
+The species list files should have been downloaded together with the rest of the `birdIndicators` repository and we can use `makeSpeciesLists()` to assemble the information. 
+
+```{r}
+sppLists <- makeSpeciesLists(Trim_data = Trim_data, spp_DataPath = "temp/birdIndicators-main/data")
```

Comment:
I cant find _Trim_data_ which is created on line 220 with eval=F but not stored anywhere that I can see.

Reply:
There, I added the review again. Not sure what happened...